### PR TITLE
A persona chapter in the Project design file

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Partially related to #1121 

* This PR creates a new chapter called Persona Creation in the project design guide

### List of changes proposed in this PR (pull-request)

* [ ] A 9 step guide to persona creation
* [ ] An image with CC-BY-SA kicense
* [ ] TOC file updated

### What should a reviewer concentrate their feedback on?

- [ ] The steps and content make sense
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by HeidiSeibold** [comment]
I am unable to review because I am chairing useR! 2020 this week and will go on holidays after :sunny: 

Sorry! :sunflower:

**Comment by malvikasharan** [comment]
I saw that on twitter. All the best with UseR 🎉💪

I have now assigned this to other Mozilla folks.
There is no hurry for the new reviewer too.

**Comment by rainsworth** [comment]
I will take a look at this now 🌟

File: book/website/project-design/persona-creation.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- Are there funding or skill-building opportunity that your persona can share more widely.
+- What other resources does your persona need to fully participate with?
+
+### Section 6: Hopes and dreams
+
+This section will reflect on the expectations, goals, and emotions of your persona in regards to this project by asking the following questions:
+
+- What does your persona want to achieve from this project?
+- What information does your persona want to gain from their participation?
+- What fear or anxiety your persona may have regarding the project?
```
Comment (rainsworth) [inline book/website/project-design/persona-creation.md:92]:
```suggestion
- What fear or anxiety may your persona have regarding the project?
```

File: book/website/project-design/persona-creation.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+This section will reflect on the hurdles and issues that your persona may encounter in regards to this project by asking the following questions:
+
+- What challenges does your persona face when participating in this project?
+- What are your persona's hurdles specific to this project?
+- How can your persona be supported in overcoming the challenges?
+- How can we surpass your persona's expectations?
+
+### Section 9: Fears and personal issues
+
+This section will reflect on fears and personal issues that your persona may face while participating in your project or community online by asking the following questions:
```
Comment (rainsworth) [inline book/website/project-design/persona-creation.md:117]:
```suggestion
This section will reflect on fears and personal issues that your persona may face while participating in this project or community online by asking the following questions:
```

File: book/website/project-design/persona-creation.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+- Would your persona be able to participate for a long period?
+- In what situations your persona might encounter difficulties during their participation?
+- What interruptions they might face while participating in this project?
+- What are the resources available to support your persona better (like funding or equipment)?
+- How can we assist your persona in reaching their goals?
+
+## Putting this together in a persona canvas
+
+The persona canvas can be used to assemble all your response in one place, share this tangible information of your mental model (abstract concepts from our thoughts) with your colleagues and create a common language to communicate about your community members, users, and contributors.
```
Comment (rainsworth) [inline book/website/project-design/persona-creation.md:127]:
```suggestion
The persona canvas can be used to assemble all your responses in one place, share this tangible information of your mental model (abstract concepts from our thoughts) with your colleagues and create a common language to communicate about your community members, users, and contributors.
```

File: book/website/project-design/persona-creation.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+The [https://www.designabetterbusiness.tools](https://www.designabetterbusiness.tools) have created multiple persona canvas (one is shown in the figure above) which are available for reuse under CC-BY ShareAlike 4.0 License, which can be downloaded [here](https://www.designabetterbusiness.tools/tools/persona-canvas).
```
Comment (rainsworth) [inline book/website/project-design/persona-creation.md:129]:
```suggestion
The [https://www.designabetterbusiness.tools](https://www.designabetterbusiness.tools) have created multiple persona canvases (one is shown in the figure above) which are available for reuse under CC-BY ShareAlike 4.0 License, which can be downloaded [here](https://www.designabetterbusiness.tools/tools/persona-canvas).
```

File: book/website/project-design/persona-creation.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+## Examples of personas in _The Turing Way_ community
+
+In the Community Handbook, we have provided a chapter on {ref}`Persona and Pathway <ch-persona>` for _The Turing Way_.
```
Comment (rainsworth) [inline book/website/project-design/persona-creation.md:133]:
```suggestion
In the Community Handbook, we have provided a chapter on {ref}`Personas and Pathways <ch-persona>` for _The Turing Way_.
```

**Comment by malvikasharan** [comment]
Thank you Rachael, for such a quick response and review. I have integrated the suggestions 💃🙏

**Pull request merged**